### PR TITLE
Allow to modify triggers collection while iterating it (prevent crash).

### PR DESCRIPTION
**Pull request by LevShisterov**

### CODE REVIEW

File: rehlds/engine/pr_cmds.cpp [hunk 1]
```diff
@@ -113,7 +113,7 @@ void EXT_FUNC SetMinMaxSize(edict_t *e, const float *min, const float *max, qboo
 	e->v.size[0] = max[0] - min[0];
 	e->v.size[1] = max[1] - min[1];
 	e->v.size[2] = max[2] - min[2];
-	SV_LinkEdict(e, 0);
```
Comment (WPMGPRoSToTeMa) [inline rehlds/engine/pr_cmds.cpp]: Maybe split refactoring changes like this into another commit?

File: rehlds/engine/pr_cmds.cpp [hunk 2]
```diff
@@ -113,7 +113,7 @@ void EXT_FUNC SetMinMaxSize(edict_t *e, const float *min, const float *max, qboo
 	e->v.size[0] = max[0] - min[0];
 	e->v.size[1] = max[1] - min[1];
 	e->v.size[2] = max[2] - min[2];
-	SV_LinkEdict(e, 0);
+	SV_LinkEdict(e, FALSE);
```
Comment (WPMGPRoSToTeMa) [inline rehlds/engine/pr_cmds.cpp]: Maybe split refactoring changes like this into another commit?

Comment (theAsmodai) [inline rehlds/engine/pr_cmds.cpp]: There is no actual need for this.
